src/mudlet.cpp

```diff
@@ -671,9 +672,6 @@ mudlet::mudlet()
     connect(this, &mudlet::signal_menuBarVisibilityChanged, this, &mudlet::slot_updateShortcuts);
     connect(this, &mudlet::signal_hostCreated, this, &mudlet::slot_assignShortcutsFromProfile);
     connect(this, &mudlet::signal_profileActivated, this, &mudlet::slot_assignShortcutsFromProfile);
-    connect(this, &mudlet::signal_tabChanged, this, [=]() {
```

Comment:
This connection is no longer needed  because it is duplicated by the one on the previous line.